app/src/main/res/layout/summing_junction_setup.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This layout file is similar to other experiments like diode, zener, etc. So we can use other experiment layout and just add a comment that reusing **which** experiment setup and re-use layout file.